src/utils.js

```diff
@@ -427,9 +430,10 @@ async function validateSecretsFile(secretsFile) {
 async function interpolateSecrets(secretsFilePath, command) {
 	try {
 		const secretsContent = await readFileContents(secretsFilePath, command, 'secrets');
+
 		// Check if environment variables are used in the file content
-		if (os.platform() === 'win32' && /(\$[a-zA-Z_][a-zA-Z0-9_]*)/.test(secretsContent)) {
-			throw new Error('Batch variables are not supported in YAML files on Windows.');
+		if (os.platform() === 'win32' && /\$({)?[a-zA-Z_][a-zA-Z0-9_]*}?/.test(secretsContent)) {
```

Comment:
Just checking, did you add tests around this?

Reply:
Done, have added tests specific to batch variables in [this commit](https://github.com/adobe/aio-cli-plugin-api-mesh/pull/137/commits/74cc4e3f7c4bf8f739ecad70dbad4c92f1fe730d)